poseidon/golang/Dockerfile

```diff
@@ -24,8 +24,7 @@ ENV PROTOC_GO_GEN_VER v1.3.2
 ENV PROTOC_GRPC_WEB_VER 1.0.7
 
 ENV BINDATA_VER v3.16.0
-ENV SQLBOILER_VER v3.6.1
-ENV SQLBOILER_CRDB_COMMIT db3049d
```

Comment:
are we not tagging specific commit id for sqlboiler-crdb/v4. This can break image on rebuild in future.